### ISSUE TITLE
command failed: dotnet build myfirstbot.csproj

### ISSUE THREAD

**Issue by singireddy777** [microsoft/BotFramework-Composer#9514]
## Describe the bug
while creating the bot by using bot framework composer ..its giving error after few steps.

## Version

Version:  2.1.2
Electron: 8.2.4
Chrome: 80.0.3987.165
NodeJS: 12.13.0
V8: 8.0.426.27-electron.0
## Browser

Mozilla firefox
- [ ] Electron distribution
- [ ] Chrome
- [ ] Safari
- [] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [] Windows
- [ ] Ubuntu

## To Reproduce

1. Go to 'bot framework composer '
2. Click on 'create new'
3. Scroll down to 'under C# empty bot' then next then bot name, location, runtime type'
4. See error: command failed: dotnet build xyz.csproj

## Expected behavior

empty  bot need to be created.
## Screenshots

Command failed: dotnet build myemptybot.csproj
## Additional context

I installed everything as mentioned in this document: https://learn.microsoft.com/en-us/composer/install-composer?tabs=windows
all are installed fine

**Comment by singireddy777** [comment]
@dmvtech ,

 Thank you for the quick reply.

I resolved this issue with the help of my colleague. The problem was I only installed .NET code SDK 3.1.426 as mentioned in the document. I didn't having installed the .NET SDK 6.0.5 version. so I installed it and the created by using BFC. hope this helps someone who facing this problem.

The combination was Node 14xxx, .NET core SDK v3.1.426 , .NET SDK v6.0.5 and Bot Framework Composer v2.1.2
